kubernetes/PROJECT

```diff
@@ -0,0 +1,7 @@
+domain: pytorch.org
```

Comment:
Just for clarification: is this the file that is passed to `kubebuilder` to generate the scaffolding for the controller that you then modified?

Reply:
It would be great if you could please add a comment to each file indicating which of these are auto-generated vs. which are written by you and meant to be edited. That way, someone in the future can only edit the files that are intended to be edited without any danger of their changes being wiped out due to codegen.

Reply:
Sure. I think developer should not touch following files.

/api/v1alpha1/groupversion_info.go
/api/v1alpha1/zz_generated.deepcopy.go
PROJECT

We are not using following configs but we may use them later. 

/config/certmanager/
/config/certmanagerprometheus
/config/webhook


Besides that, we expect developer to contribute to `api/v1alpha1/` and `/controllers`. Definitely they may need to touch Makefile or go.mod.